app/src/components/cards/AccountDetailsCard/AccountDetailsCard.tsx

```diff
@@ -190,11 +198,11 @@ const HashExpandWrapper = styled.div`
   margin: auto 0;
 `;
 
-const HashHeading = styled(Heading)<{
+const HashHeading = styled.div<{
```

Comment:
Does `styled(Heading)` still not work here?

Reply:
It does now.  Updated!